### ISSUE TITLE
Live Preview Window not appearing

### ISSUE THREAD

**Issue by haroldsoh** [lloeki/svg-preview#16]
I cannot see the SVG Preview window. Am on Atom v1.03, Ubuntu Linux 14.04 64bit with a samples.svg file open: 
```svg
<svg version="1.1"
  viewbox="0 0 100 100"
  xmlns="http://www.w3.org/2000/svg">

  <g display="inline">
  <line  stroke-width="2" stroke="black" x1="0" x2="100" y1="100" y2="0"/>
  </g>
</svg>
```
I've tried toggling the window (Ctrl-Alt-V, Alt-V, Cmd-Alt-V) and via the Menu bar with no success.

**Comment by lloeki** [comment]
I take it you're using svg-preview together with language-svg. This is fixed already on master, I just forgot to make a release. It's now included in v0.6.1.

**Comment by haroldsoh** [comment]
Works great after updating. Thanks!
